src/patternfly/components/TreeView/tree-view-list-item.hbs

```diff
@@ -0,0 +1,8 @@
+<li class="pf-c-tree-view__list-item{{#if tree-view-list-item--expandable}} pf-m-expandable{{/if}}{{#if tree-view-list-item--expanded}} pf-m-expanded{{/if}}{{#if tree-view-list-item--modifier}} {{tree-view-list-item--modifier}}{{/if}}"
```

Comment:
I know we have a mix of use of attrs/vars in other components, but do you think for these boolean ones, they should be `tree-view-list-item--IsExpandable` and `tree-view-list-item--IsExpanded`?